### PR TITLE
Update Customize the top side of the portfolio page

### PR DESCRIPTION
**Pull request by rica213**
Hey code reviewer,
This is my current PR. I created it because I accidentally merge the branch on my local machine. Afterward, I could not push on the remote branch and the only one solution was to open a new PR. I have already fixed all the required changes from the previous reviews.
After troubleshooting with a code reviewer they allowed me to open a new PR since we were not able to push on the previous PR but here you can see the current PR with all request changes from the previous reviews.
[Kindly have a look on the current PR](https://github.com/rica213/portfolio/pull/1)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      </h1>
+      <p id="intro">
+        Hello! I am a software developer! I can help you build a product,
+        feature or website. Take a look of my works. If you like what you see
+        and have a project you need coded, don’t hesitate and contact me.
+      </p>
+      <a href="#" id="collaboration">Start collaboration</a>
+      <div id="separator">1</div>
+    </section>
+  </body>
```
Comment (Felix45) [inline index.html:44]: - [ ] Implemented all the changes from the previous reviewer :heavy_check_mark: 

Screenshot from 2022-12-02 16-57-46

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle: 
Hi @rica213 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights :heavy_check_mark: 
- [ ] Implemented all the changes from the previous reviewer :white_check_mark: 

Congratulations-Memes-FEATURED

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rica213** [comment]
Thank you all for your time to review my code. @Felix45 @Alfredbis29

**Comment by rica213** [comment]
.

**Comment by Alfredbis29** [comment]
no problem @rica213  Good luck :+1:

**Pull request merged**